Traceback (most recent call last):
  File "c:\Users\40020507\Downloads\diagslave-3.4\win\new.py", line 11, in <module>
    server_socket.bind((SERVER_IP, SERVER_PORT))
OSError: [WinError 10049] The requested address is not valid in its context
